metric.go

```diff
@@ -1,89 +1,78 @@
-// Copyright 2014 Prometheus Team
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package main
+package prom2json
 
 import (
-	"crypto/tls"
-	"encoding/json"
-	"flag"
 	"fmt"
 	"io"
 	"mime"
 	"net/http"
-	"os"
-	"runtime"
 
 	"github.com/matttproud/golang_protobuf_extensions/pbutil"
 	"github.com/prometheus/common/expfmt"
-	"github.com/prometheus/common/log"
+	"github.com/prometheus/log"
 
 	dto "github.com/prometheus/client_model/go"
+
 )
 
 const acceptHeader = `application/vnd.google.protobuf;proto=io.prometheus.client.MetricFamily;encoding=delimited;q=0.7,text/plain;version=0.0.4;q=0.3`
 
-type metricFamily struct {
+// Family holds the s group of metrics/summary/histogram of a certain name...
+type Family struct {
+	//Time    time.Time
 	Name    string        `json:"name"`
 	Help    string        `json:"help"`
 	Type    string        `json:"type"`
 	Metrics []interface{} `json:"metrics,omitempty"` // Either metric or summary.
 }
 
-// metric is for all "single value" metrics.
-type metric struct {
+// Metric is for all "single value" metrics.
+type Metric struct {
 	Labels map[string]string `json:"labels,omitempty"`
 	Value  string            `json:"value"`
 }
 
-type summary struct {
+// Summary describes a summary metric
```

Comment:
`// Summary mirrors the Summary proto message.`

(Note that sentences should end with a period.)